### ISSUE TITLE
APOC MongoDB procedures hard to source and install on windows without MAVEN

### ISSUE THREAD

**Issue by miConsent** [neo4j-contrib/neo4j-apoc-procedures#265]
I am unable to obtain the correct jar files to take advantage of the APOC MongoDB integration. I am running NEO4J 3.1.1 Enterprise on Windows.

The documentation says 

> "Download latest release
> Go to http://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/latest to find the latest release and download the binary jar to place into your $NEO4J_HOME/plugins folder."

That's understood - I actually needed 3.1.0.3 to work with Neo4j 3.1.1 (but I got there)

> "Copy these jars into the plugins directory:
> mvn dependency:copy-dependencies
> cp target/dependency/mongodb*.jar target/dependency/bson*.jar $NEO4J_HOME/plugins/"

Without a maven environment I have no way of determining what those jar files are or where to source them from. 

It would be helpful to provide a list of what Jar files are required,  links in the documentation to where they may be found,  or an easy to use alternative method that does not require complex tooling to obtain.

Cheers

**Comment by miConsent** [comment]
I Managed to work out a solution that others may want to know how to do for themselves.

I now have all of these JAR files in my plugins directory; I am not entirely sure which ones are of actual use as there seem to be three actual mongo drivers, as sync, async and java driver. Presumably the java driver is the actual one used.

> apoc-3.1.0.3-all.jar
> bson-3.4.2.jar
> mongo-java-driver-3.4.2-javadoc.jar
> mongo-java-driver-3.4.2-sources.jar
> mongo-java-driver-3.4.2.jar
> mongodb-driver-3.4.2-javadoc.jar
> mongodb-driver-3.4.2-sources.jar
> mongodb-driver-3.4.2.jar
> mongodb-driver-async-3.4.2-javadoc.jar
> mongodb-driver-async-3.4.2-sources.jar
> mongodb-driver-async-3.4.2.jar
> mongodb-driver-core-3.4.2-javadoc.jar
> mongodb-driver-core-3.4.2-sources.jar
> mongodb-driver-core-3.4.2.jar

I sourced all of the mongo*.jar files from here ; [here](https://mongodb.github.io/mongo-java-driver/)

And selecting all of the driver types at version 3.4.2 and downloading all of the resulting JAR files from the lists it gave me.

I also found the bson*.jar files via the download button  [here](https://mvnrepository.com/artifact/org.mongodb/bson/3.4.2)

With all of these jar files in the plugins folder of each node and restarting the cluster I now get a response from Mongo! Cool.

Now for some graph development with Mongo as a side kick!
